stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -87,7 +87,8 @@ public enum _RuntimeConfig {
 
   /// When true, run the entire tensor computation in
   /// _TFCStartTensorComputation(), instead of running it on a separate thread.
-  /// - Note: Set to true only for debugging purposes.
+  /// - Note: Set to true only for debugging purposes, as it has limited
+  /// functionality (e.g. no sends/recvs support).
```

Comment:
Start of a new line in a doc comment item should align with the name of that item, i.e. "Note".